.travis.yml

```diff
@@ -7,9 +7,16 @@ python:
   - "pypy"
 
 install:
-  - "python setup.py --version"
-  - "travis_retry pip install -r requirements.txt"
-  - echo ${TRAVIS_PYTHON_VERSION}
+  - travis_retry pip install -r requirements.txt
+  - travis_retry pip install python-coveralls coverage nose-cov
+  - echo TRAVIS_PYTHON_VERSION=${TRAVIS_PYTHON_VERSION}
   - if [ ${TRAVIS_PYTHON_VERSION:0:1} != "3" ]; then travis_retry pip install contextlib2; fi
+  - python setup.py --version
 
-script: nosetests
+# Run test
+script:
+  - nosetests -v --with-cov --cov streamparse --cov-config .coveragerc --logging-level=DEBUG
+
+# Calculate coverage on success
+after_success:
+  - coveralls --config_file .coveragerc
```

Comment:
3.4 sounds good to me.  The only reason they would be different is if we had different code paths based on the Python version, but we don't have much of that.  For SKLL, the 2.7 and and 3.x versions due do a couple things rather differently, so we record coverage for both, duplicate comments be damned.